src/status_im/ui/screens/wallet/main/views.cljs

```diff
@@ -131,11 +134,18 @@
     [add-asset]))
 
 (defn asset-section [balance prices-loading? balance-loading?]
-  (let [assets [{:id "eth" :currency :eth :amount balance}]]
+  (let [assets (concat [{:id "eth" :currency :eth :amount balance}]
+                       (if config/erc20-enabled?
+                         [{:id "snt" :currency :snt :amount 5000000000000000000000}]))]
     [react/view {:style styles/asset-section}
      [react/text {:style styles/asset-section-title} (i18n/label :t/wallet-assets)]
      [list/flat-list
-      {:data       (conj assets {}) ;; Extra map triggers rendering for add-asset
+      {:data       (concat assets [{}]) ;; Extra map triggers rendering for add-asset
```

Comment:
Btw @goranjovic I realized this can also be put under flag, especially since there seems to be some comments here. So something like `{:data (if erc20 (conc..) (conj...)) ...`